core/common/assets/js/utils/helpers.js

```diff
@@ -37,41 +37,4 @@ export default class Helpers {
 			return $1.toUpperCase();
 		} );
 	}
-
-	findViewRecursive( parent, key, value, multiple = true ) {
```

Comment:
add `elementorCommon.helpers.softDeprecated` message.

Reply:
is new and never used before.